src/node_file.cc

```diff
@@ -885,6 +885,56 @@ static void WriteBuffer(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+// Wrapper for writev(2).
+//
+// bytesWritten = writev(fd, chunks, position, callback)
+// 0 fd        integer. file descriptor
+// 1 chunks    array of buffers to write
+// 2 position  if integer, position to write at in the file.
+//             if null, write from the current position
+static void WriteBuffers(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (!args[0]->IsInt32())
```

Comment:
Right, I kinda went with what I saw in `WriteBuffer`, which is located right above my `WriteBuffers` function. It too has a test like:

```c++
  if (!args[0]->IsInt32())
    return env->ThrowTypeError("First argument must be file descriptor");
```